src/Connection.php

```diff
@@ -67,6 +101,30 @@ public function getChannel()
         return $this->channel;
     }
 
+    /**
+     * Fetch the list of consumers details for the given queue name using the management API.
+     *
+     * @param  string $queueName
+     * @param  string $columns
+     *
+     * @return array
+     */
+    public function fetchQueueConsumers($queueName, string $columns = 'consumer_details.consumer_tag')
+    {
+        $http = app(HTTPClient::class);
```

Comment:
Is this testable?

Reply:
Where is the test?

Reply:
Done 
https://github.com/Vinelab/bowler/commit/e06cc2fcd72134ad7035304e94646af4ef0a0a32